packages/fuel-indexer-macros/src/decoder.rs

```diff
@@ -833,7 +833,7 @@ impl From<ObjectDecoder> for TokenStream {
                                             .map(|query| query.to_string())
                                             .collect::<Vec<_>>();
 
-                                        d.lock().await.put_many_to_many_record(queries).await;
+                                        d.lock().await.put_many_to_many_record(queries).await.expect(&format!("Entity::save_many_to_many for {} failed.", stringify!(#ident)));
```

Comment:
- I think here we should be logging the error messages (and `.expect`) as close to the actual culprit call as possible
- In this case the culprit of this error would be `fuel_indexer::database::Database:: put_many_to_many_record`
  - If the error is logged/handled there, we don't need to handle it here